test/MockWebServer_IntegrationTest.php

```diff
@@ -268,4 +268,12 @@ public function requestInfoProvider() {
 		];
 	}
 
+	public function testServerIsRunning() {
+		$this->assertTrue(self::$server->isRunning());
+	}
+
+	public function testStopServer() {
+		self::$server->stop();
+		$this->assertFalse(self::$server->isRunning());
+	}
```

Comment:
I think this test should probably create it's own instance of MockWebServer rather than use the class wide instance. Running the tests out of order would break this.